packages/@ionic/cli-utils/src/lib/project/angular/generate.ts

```diff
@@ -208,8 +209,8 @@ ${chalk.cyan('[1]')}: ${chalk.bold('https://ionicframework.com/docs/cli/projects
   private async getSchematics(): Promise<Schematic[]> {
     if (!this.schematics) {
       try {
-        const { SchematicEngine } = await import('@angular-devkit/schematics');
-        const { NodeModulesEngineHost } = await import('@angular-devkit/schematics/tools');
+        const { SchematicEngine } = await importAppScripts('@angular-devkit/schematics');
```

Comment:
You'll need to pass in `this.project.directory` to the two functions, not the module name.